src/Symfony/Component/VarExporter/Tests/LazyGhostObjectTraitTest.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarExporter\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\VarExporter\Internal\GhostObjectRegistry;
+use Symfony\Component\VarExporter\Tests\Fixtures\LazyGhostObject\ChildMagicClass;
+use Symfony\Component\VarExporter\Tests\Fixtures\LazyGhostObject\ChildTestClass;
+use Symfony\Component\VarExporter\Tests\Fixtures\LazyGhostObject\MagicClass;
+use Symfony\Component\VarExporter\Tests\Fixtures\LazyGhostObject\TestClass;
+
+class LazyGhostObjectTraitTest extends TestCase
+{
+    public function testGetPublic()
+    {
+        $instance = ChildTestClass::createLazyGhostObject(function ($ghost) {
```

Comment:
I think it would be helpful to add type declaration for $ghost param. This way you get autocomplete and help analysers.